nucliadb/nucliadb/ingest/orm/brain.py

```diff
@@ -374,6 +374,7 @@ def set_global_tags(self, basic: Basic, uuid: str, origin: Optional[Origin]):
                         relation=Relation.COLAB,
                         source=relationnodedocument,
                         to=relationnodeuser,
+                        relation_label=RelationTypePbMap[Relation.COLAB].value,
```

Comment:
Isnt it redundant ?

Reply:
Yes and no. It's a way to label the automatic relations, as they are going to presented in the API, and avoid returning empty strings. If it's not desired, I can remove them.

Reply:
Yes, the entity relation type already provides this information. We can change in the future if its required